src/libotutil/ot-variant-builder.c

```diff
@@ -841,15 +847,16 @@ ot_variant_builder_info_free (OtVariantBuilderInfo *info)
 
 OtVariantBuilder *
 ot_variant_builder_new (const GVariantType *type,
-                        int fd)
+                        int fd,
+                        GError **error)
 {
   OtVariantBuilder *builder;
 
   g_return_val_if_fail (g_variant_type_is_container (type), NULL);
 
   builder = (OtVariantBuilder *) g_slice_new0 (OtVariantBuilder);
 
-  builder->head = ot_variant_builder_info_new (builder, type);
+  builder->head = ot_variant_builder_info_new (builder, type, error);
```

Comment:
Ugh, I actually forgot to error-check here, as well as dropping the `g_return_val_if_fail` above.